usage-examples/test-execution.md

```diff
@@ -0,0 +1,42 @@
+# Test Execution Example
+This example presents how Eiffel events in the test domain can be connected.
+
+## Introduction
+Using Eiffel events to describe the execution of a test activity can be done in several ways. This example shows how to make use of several non-mandatory events, with the purpose to present how they all are related.
+
+## Event Graph
+![alt text](./test-execution.png "Event Graph of Test Execution Example")
+
+## Event-by-Event Explanation
+### ActT1, ActS1, ActF1
+[EiffelActivityTriggeredEvents](../eiffel-vocabulary/EiffelActivityTriggeredEvent), [EiffelActivityStartedEvents](../eiffel-vocabulary/EiffelActivityStartedEvent) and [EiffelActivityFinishedEvents](../eiffel-vocabulary/EiffelActivityFinishedEvent), in this example representing build activities. Using its __CONTEXT__ link, the [EiffelArtifactCreatedEvents](../eiffel-vocabulary/EiffelArtifactCreatedEvent.md) __ArtC1__ and __ArtC2__ declare that they are part of the activity.
+
+### TERC1
```

Comment:
I think the "1" is superfluous, since there's only one iteration in the example. Same goes for all other events, e.g. "EDef1A" could be just "EDef A", unless you plan to add additional iterations in the example?